docs/security/simple-ldap.md

```diff
@@ -1,42 +1,124 @@
+
+For either method, the database server rejects the connection if the client user name and the host name do not match a server account.
+
+If a database server LDAP authentication is successful, the LDAP server searches for an entry. The LDAP server matches the user and authenticates using the LDAP password. If the database server account names the LDAP user distinguished name (DN), added by the [`BY clause`](#use-the-by-clause-that-specifies-the-authentication-string), the LDAP server uses that value and the LDAP password provided by the client. This method fails in the following conditions:
+
+* DN and password have incorrect values
+* DN and password are empty and the LDAP server does not permit anonymous connections
+
+If the LDAP server finds multiple matches or no match, authentication fails.
+
+If the password is correct, and the LDAP server finds a match, then LDAP authentication succeeds. The LDAP server returns the LDAP entry and the authentication plugin determines the authenticated user's name based on the entry. If the LDAP entry has no group attribute, the plugin returns the client user name as the authenticated name. If the LDAP entry has a group attribute, the plugin returns the group value as the authenticated name.
+
+The database server compares the client user name to the authenticated user name. If these names are the same, the database server uses the client user name to check for privileges. If the name differs, then the database server looks for an account that matches the authenticated name.
+
+## Prerequisites for authentication
+
+The LDAP authentication plugins required the following:
+
+* An available LDAP server
+
+* The LDAP server must contain the LDAP user accounts to be authenticated
+
+* The OpenLDAP client library must be available on the same system as the plugin
+
+The SASL-based LDAP authentication additionally requires the following:
+
+* Configure the LDAP server to communicate with a SASL server
+
+* Available SASL client library on the same system as the client plugin. 
+
+* Services are configured to use the supported SCRAM-SHA-1 SASL mechanism 
+
+## Install the plugins
+
+You can use either of the following methods to install the plugins.
+
+The SASL-based LDAP authentication is available on *Percona Server for MySQL* 8.0.30-22 and later.
+
+
+### Load the plugins at server start
+
+Add the following statements to your `my.cnf` file:
+
+```text
+[mysqld]
+plugin-load-add=authentication_ldap_simple.so
+authentication_ldap_simple_server_host=127.0.0.1
+authentication_ldap_simple_bind_base_dn='dc=percona, dc=com'
+plugin-load-add=authentication_ldap_sasl.so
+authentication_ldap_sasl_server_host=127.0.0.1
```

Comment:
this example mixes simple and sasl plugins

Reply:
Fixed.